### PR TITLE
Update UG and fix Autocomplete

### PR DESCRIPTION
**Pull request by KS2499**
### Changes
- Fix `Autocomplete` by reordering `DeleteCommand` and `DeleteMeetingCommand`
- Update UG with page-breaks for PDF
- Update `Autocomplete` Note in UG
- Update `CommandSummary` table in UG
- Update `help` command instructions in UG
- Fix some typo and formatting

### Here is the UG in PDF format for your review

[User Guide v3.pdf](https://github.com/AY2223S2-CS2103T-T11-2/tp/files/11189926/User.Guide.v3.pdf)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -298,9 +304,9 @@ Examples:
 
 **:information_source: Note:**<br>
 
-Autocomplete only works for command words and prefixes that expect a corresponding input data. You may refer to the
-[Prefx](#prefix) section for more details on prefixes.
-
+* Autocomplete only works for command words and prefixes that expect a corresponding input data. You may refer to the
+[Prefix](#prefix) section for more details on prefixes.<br>
+* If input matches a command, Autocomplete will autocomplete the [prefixes](#prefix) for the command. If command requires no prefix, autocomplete will not do anything else.
```
Comment (Beebeeoii) [inline docs/UserGuide.md:309]:
```suggestion
* If input matches a command, Autocomplete will autocomplete the [prefixes](#prefix) for the command. If the command does not require any prefix, autocomplete will not do anything.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -298,9 +304,9 @@ Examples:
 
 **:information_source: Note:**<br>
 
-Autocomplete only works for command words and prefixes that expect a corresponding input data. You may refer to the
-[Prefx](#prefix) section for more details on prefixes.
-
+* Autocomplete only works for command words and prefixes that expect a corresponding input data. You may refer to the
+[Prefix](#prefix) section for more details on prefixes.<br>
```
Comment (Beebeeoii) [inline docs/UserGuide.md:308]: do we need the `<br>`? wouldnt point form (`*`) already break lines automatically?

**Review by Beebeeoii — APPROVED** [review]
minor nits, if not lgtm !

File: docs/UserGuide.md (hunk 2 above)
Comment (KS2499) [reply docs/UserGuide.md]: Yah true, think I forgot to remove it. Removed

**Review by Beebeeoii — APPROVED** [review]

**Pull request merged**